README.md

```diff
@@ -19,7 +19,14 @@ Thank you!
 
 ## Prettifying
 
-`indent --k-and-r-style --use-tabs --tab-size4 --braces-on-if-line --cuddle-else --dont-break-function-decl-args --line-length120 --swallow-optional-blank-lines apps/app_rpt.c`
+Note: This is an optional step. If you are touching existing code in a few places, then it is best to skip prettifying and just let the commit checker check for conformance to the coding standards. 
+
+If `clang-format` isn't already installed, install `clang-format` on your development system.  
+
+From the top level project directory run:
+
+`clang-format -style file:.dev/.clang-format your_file > your_prettified_file`
+
 
```

Comment:
"maybe" say something about setting up a pre-commit hook?

After installing `clang-format` and `codespell`, set up the pre-commit workflow as follows:
From the top level project directory, execute:
 `cp ./.dev/pre-commit ./.git/hooks/pre-commit`

This will "pre-check" your commit against the repository workflow reducing the number of commits.